keel-bakery-plugin/src/main/kotlin/com/netflix/spinnaker/keel/bakery/resource/ImageHandler.kt

```diff
@@ -84,6 +84,18 @@ class ImageHandler(
     resource: Resource<ImageSpec>,
     resourceDiff: ResourceDiff<Image>
   ): List<Task> {
+
```

Comment:
Did you put this in here (vs where we detect the diff) because the general diff detection logic is generic, so there's no place for this kind of "ignore the diff in this situation" logic?

Reply:
I think I was working in the old mindset of the `current` method should return the actual current state of the world, but in hindsight I'd be better overriding desired to trim the set of regions

Reply:
Ya I think that would be slightly better

Reply:
I concur

Reply:
Updated to modify the current state call instead